lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -536,7 +536,37 @@ void Interpreter::fwdSigmoidInst(const SigmoidInst *I) {
   }
 }
 
+/// Calculate quantized tanh.
```

Comment:
I would name this method int8Tanh, or tableInt8Tanh. Technically every method is "calculate"-something.